tasklist/els-schema/src/main/java/io/camunda/tasklist/schema/indices/AbstractIndexDescriptor.java

```diff
@@ -47,6 +45,31 @@ public String getSchemaClasspathFilename() {
 
   @Override
   public String getAllVersionsIndexNameRegexPattern() {
-    return String.format("%s-%s-\\d.*", tasklistProperties.getIndexPrefix(), getIndexName());
+    return formatAllVersionsIndexNameRegexPattern(getIndexPrefix(), getIndexName());
+  }
+
+  public static String formatFullQualifiedIndexName(
+      final String indexPrefix, final String indexName, final String version) {
+    return String.format(
+        "%s%s-%s-%s_", formatIndexPrefix(indexPrefix), TASK_LIST, indexName, version);
```

Comment:
🔧 Should we maybe make this format `"%s%s-%s-%s_"` global accessible (static const) like the formatIndexPreifx, and reuse this everywhere?